packages/experimental/MenuButton/src/MenuButton.tsx

```diff
@@ -43,6 +43,7 @@ export const MenuButton = compose<MenuButtonType>({
       disabled,
       primary,
       icon,
+      style,
```

Comment:
We don't need to do something fancy with merging the style generated by tokens with the user passed in style?

Reply:
This is a scenario where I wasn't sure what we're actually supposed to do.. so I'm not sure I can answer the question myself.